lua/toggleterm.lua

```diff
@@ -222,16 +222,23 @@ end
 -- If any terminal is open it will be closed
 -- If no terminal exists it will do nothing
 -- If any terminal exists but is not open it will be open
-function M.toggle_all()
+function M.toggle_all(force)
   local ui = require("toggleterm.ui")
   local terminals = terms.get_all()
-  if not ui.find_open_windows() then
+
+  if force == true and ui.find_open_windows() then
```

Comment:
```suggestion
  if force and ui.find_open_windows() then
```

Reply:
You're right, it's redundant and gone. Thanks for the sharp look.